index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="index.css">
+  </head>
+
+  <body>
+    <div id="main" class="align">
+
+<h1 id="title"> Dr. Norman Borlaug</h1>
+<p id="tribute-info">The man who saved a billion lives</p>
+<div id="img-div">
+  <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" alt="Dr. Norman Borlaug" id="image">
+<p id="image-caption">Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.</p>
+</div>
+<a href="https://en.wikipedia.org/wiki/Norman_Borlaug" id="tribute-link" target="_blank"></a>
+    </div>
+  </body>
+</html>
```

Comment:
Please format your document, so that it would be easier for other developers or reviewers to read. I recommend installing 'Prettier-Code formatter', it would help your code to be cleaner & easier to read in addition to making it seem more professional.